packages/elcollectooorr/__init__.py

```diff
@@ -18,3 +18,7 @@
 # ------------------------------------------------------------------------------
 
 """This module contains the elcollectooorr packages authored by Valory AG."""
+from pathlib import Path
+
+
+ELCOLLETOOORR_PACKAGES = Path(__file__).parent
```

Comment:
No you cannot do this. It won't exist outside the context of this repo, so your tests will break.

Reply:
why? evertime an elcollectoorr package is there, the elcollectoorr author will also be there. Are `__init__.py` synced?